themes/cardinal_service/lib/components/opportunity-row/opportunity_row.ui_patterns.yml

```diff
@@ -54,11 +54,16 @@ opportunity_row:
       label: 'Open To'
       description: 'Category of student of the opportunity.'
       preview: '<ul><li>Everyone</li></ul>'
+    program:
+      type: text
+      label: 'Program'
+      description: 'Opportunity Program.'
```

Comment:
Did you forget to export the configuration to map the opportunity display to this new field?